src/main/kotlin/org/rust/cargo/project/settings/ui/RustProjectSettingsPanel.kt

```diff
@@ -98,6 +104,7 @@ class RustProjectSettingsPanel(private val cargoProjectDir: String = ".") : Disp
         if (PlatformUtils.isIntelliJ()) {
             row("Use cargo check:") { useCargoCheckForBuildCheckbox() }
         }
+        row(label = Label("Use cargo check to analyze code:")) { useCargoCheckAnnotatorCheckbox() }
```

Comment:
Having two settings with nearly identical names is a bit confusing. Let's rename "Use cargo check:" to "Use cargo check when build project:"
@matklad?